source/GameData.cpp

```diff
@@ -797,43 +836,21 @@ void GameData::LoadSources()
 	for(const string &path : globalPlugins)
 	{
 		if(Files::Exists(path + "data") || Files::Exists(path + "images") || Files::Exists(path + "sounds"))
-			sources.push_back(path);
+			if(LoadAboutPlugin(path))
+				sources.push_back(path);
 	}
 
 	vector<string> localPlugins = Files::ListDirectories(Files::Config() + "plugins/");
 	for(const string &path : localPlugins)
 	{
 		if(Files::Exists(path + "data") || Files::Exists(path + "images") || Files::Exists(path + "sounds"))
-			sources.push_back(path);
+			if(LoadAboutPlugin(path))
+				sources.push_back(path);
```

Comment:
```
	if((Files::Exists(path + "data") || Files::Exists(path + "images") || Files::Exists(path + "sounds"))
			&& LoadAboutPlugin(path))
		sources.push_back(path);
```
?

Reply:
I made this change in two places and attributed you.